geoserve/geoserve.go

```diff
@@ -13,26 +13,28 @@ import (
 
 	"github.com/getlantern/golog"
 	"github.com/golang/groupcache/lru"
+	"github.com/mholt/archiver/v3"
 	geoip2 "github.com/oschwald/geoip2-golang"
 )
 
 const (
-	DB_URL = "https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz"
+	DB_URL = "https://download.maxmind.com/app/geoip_download?license_key=%s&edition_id=GeoLite2-City&suffix=tar.gz"
```

Comment:
See https://dev.maxmind.com/geoip/geoipupdate/#Direct_Downloads. The `suffix` param can't be omitted so I left it as is.